logging/fields/fields.go

```diff
@@ -195,23 +208,10 @@ func Count(val int) zap.Field {
 	return zap.Int(FieldCount, val)
 }
 
-func ErrorStrs(val []string) zap.Field {
-	return zap.Any(FieldErrors, val)
-}
-
-func Errors(val []error) zap.Field {
-	return zap.Any(FieldErrors, val)
-}
-
 func Topic(val string) zap.Field {
 	return zap.String(FieldTopic, val)
 }
 
-func DutyID(runner runner.Runner, duty *spectypes.Duty) zap.Field {
-	return zap.Stringer(FieldDutyID, funcStringer{
-		fn: func() string {
-			epoch := runner.GetBaseRunner().BeaconNetwork.EstimatedEpochAtSlot(duty.Slot)
-			return fmt.Sprintf("T:%v::E:%v::S:%v::V:%v", duty.Type.String(), epoch, duty.Slot, duty.ValidatorIndex)
-		},
-	})
+func ConsensusTime(val time.Time) zap.Field {
```

Comment:
Computing durations on the logging side is not ideal, it's better to pass the already computed duration on the caller's side, because it's more predictable & precise

So just `func ConsensusTime(dur time.Duration)` would be enough 👍

Also, sometimes you wanna stop counting the duration now, but only log it later, which is impossible if you have to pass time.Time